### ISSUE TITLE
got 400: bad request

### ISSUE THREAD

**Issue by tarokiritani** [jupyter/nbviewer#664]
My jupyter notebook renders ok on github, but not on nbviewer. I got error "400: Bad Request".

My notebook is here:

https://github.com/tarokiritani/in-vivo-wcr/blob/master/electrophysiology.ipynb

**Comment by parente** [comment]
The notebook is stored in Git LFS. It's likely the current nbviewer implementation doesn't support fetching from that.

**Comment by tarokiritani** [comment]
Yes, it is a very large (>100 MB) notebook. Thanks a lot for looking into the issue.

**Comment by Honghe** [comment]
@parente Does nbviewer have plan to support Git LFS ipynb?

**Comment by banesullivan** [comment]
+1 for LFS support.

This is needed with the recent support for rendering `panel` outputs
